project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.11.0-M2")
\ No newline at end of file
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.11.0")
```

Comment:
when running `g8Test` multiple times and importing the project to intelliJ i get:
![image](https://user-images.githubusercontent.com/7117869/55488638-b5cc6100-5630-11e9-8fa2-31bfb7f29f01.png)

Reply:
IntelliJ adds its files to target and makes g8 crazy. Should work fine for regular usage.